### ISSUE TITLE
Extending to French 

### ISSUE THREAD

**Issue by cring031** [akoumjian/datefinder#131]
Hello akoumjian and the datefinder community,
i wanted to use your software brick for a academic work,
your  library is excellent for english, but i have some trouble to extend it to French.

I added in the constant file french months and days, but i could just extract years contained in a text for the moment...
For example on this wikipage  : https://fr.wikipedia.org/wiki/Giovanni_Soranzo i wanna catch the birth and death dates in the first introductive paragraph, but i just get the years included inside this text. 

Have you some advice for me ? I had the feeling that i have to declare somewhere else some patterns, can you help me on that ? 
(i will for sure pushing all the fixes that i made on the git :) ) 

Thank you by advance !

**Comment by rth** [comment]
@cring031 Could you please share the constants file you have for French? I would be interested in it as well.

**Comment by hzitoun** [comment]
yes please @cring031, is there any possibility to share the file?

**Comment by cring031** [comment]
Hey guys ! Fine to have some news, i get you here "french patterns" in the same way than you put it in the constant file.
An important thing to know that i didn't write here is that the french date template are more writing by the year "dd-MM-YYY" or "dd-MM-YYYY patterns. Your package could be extremely promising !  Maybe that you will have to implement a langage detector for handling more finely theses "linguistic specificities"....

NUMBERS_PATTERN = r"premier|deux|trois|quatre|cinq|six|sept|huit|neuf|dix|onze|douze|treize|quatorze|quinze|seize|dix-sept|dix-huit|dix-neuf|vingt|vingt-et-un|vingt-deux|vingt-trois|vingt-quatre|vingt-cinq|vingt-six|vingt-sept|vingt-huit|vingt-neuf|trente|trente-et-un"
#POSITIONNAL_TOKENS = r"prochain|dernier"
DIGITS_PATTERN = r"\d+"
DIGITS_SUFFIXES = r"er|e"
DAYS_PATTERN = "lundi|mardi|mercredi|jeudi|vendredi|samedi|dimanche|lun\.?|mar\.?|mer\.?|jeu\.?|ven\.?|sam\.?|dim\.?"
MONTHS_PATTERN = r"janvier|fevrier|mars|avril|mai|juin|juilet|août|septembre|octobre|novembre|decembre|janv\.?|févr\.?|avr\.?|juil\.?|juill\.?|sept\.?|oct\.?|nov\.?|déc\.?"
RELATIVE_PATTERN = "avant|après|prochain|dernier|il y a"
TIME_SHORTHAND_PATTERN = "aujourd'hui|hier
UNIT_PATTERN = "seconde|minute|heure|jour|semaine|mois|année"

**Comment by akoumjian** [comment]
Maintainer triage note: this comment/action is being posted by Codex AI agent at Alec Koumjian's direction.

Keeping this issue open as a post-rc3 enhancement request.

GA blocker status: not a blocker for v1.0.0 GA.

Thank you for the idea and contribution.
